src/Microsoft.ML.Ensemble/OutputCombiners/MultiStacking.cs

```diff
@@ -38,13 +41,28 @@ private static VersionInfo GetVersionInfo()
         [TlcModule.Component(Name = LoadName, FriendlyName = Stacking.UserName)]
         public sealed class Arguments : ArgumentsBase, ISupportMulticlassOutputCombinerFactory
         {
+            [Argument(ArgumentType.Multiple, HelpText = "Base predictor for meta learning", ShortName = "bp", SortOrder = 50,
+                Visibility = ArgumentAttribute.VisibilityType.CmdLineOnly, SignatureType = typeof(SignatureMultiClassClassifierTrainer))]
+            [TGUI(Label = "Base predictor")]
+            public IComponentFactory<ITrainer<IPredictorProducing<VBuffer<Single>>>> BasePredictorType;
+
+            public override IComponentFactory<ITrainer<TVectorPredictor>> BasePredictorFactory
+            {
+                get { return BasePredictorType; }
+                set { BasePredictorType = value; }
+            }
```

Comment:
Hmmm. It seems like this boilerplate is necessarily only because `ArgumentAttribute` cannot be applied to properties, so we have to declare this backing field, which is then exposed through the property. If that is so, maybe we ought to just allow it to be applied to properties and change the command line parsing code to handle properties as well as fields, so we can avoid this awkwardness.

Reply:
As we discussed offline, I changed this to `internal`, since Stacking doesn't allow external inheritors anyway. I also changed it to a method, since the base class only needs to retrieve the component factories.